### PR TITLE
py-bidscoin: add v4.1.1 and py-argparse-manpage: add new package

### PR DESCRIPTION
**Pull request by manuelakuhn**
https://github.com/Donders-Institute/bidscoin/tree/4.1.1
https://github.com/praiskup/argparse-manpage/tree/v4.5

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the new package's home page and all affected version sha256.

**Review by adamjstewart — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-bidscoin/package.py [hunk 1]
```diff
@@ -13,27 +13,34 @@ class PyBidscoin(PythonPackage):
...
+    depends_on("py-argparse-manpage+setuptools", when="@4.1:", type="build")
 
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-pydicom@2:", type=("build", "run"))
-    depends_on("py-pyqt5@5.12.1:", type=("build", "run"))
+    depends_on("py-pyqt6", when="", type=("build", "run"))
     depends_on("py-ruamel-yaml@0.15.35:", type=("build", "run"))
+    depends_on("py-tomli@1.1:", when="@4: ^python@:3.10", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bidscoin/package.py]: Not in 4.0.0

File: var/spack/repos/builtin/packages/py-bidscoin/package.py [hunk 2]
```diff
@@ -13,27 +13,34 @@ class PyBidscoin(PythonPackage):
...
     depends_on("py-setuptools", type="build")
-    depends_on("py-pytest-runner", type="build")
+    depends_on("py-argparse-manpage+setuptools", when="@4.1:", type="build")
 
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-pydicom@2:", type=("build", "run"))
-    depends_on("py-pyqt5@5.12.1:", type=("build", "run"))
+    depends_on("py-pyqt6", when="", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bidscoin/package.py]: when?

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by manuelakuhn** [comment]
Thank for cross-checking!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**